beacon-chain/state/fieldtrie/field_trie_helpers.go

```diff
@@ -328,3 +330,31 @@ func handleBalanceMVSlice(mv multi_value_slice.MultiValueSliceComposite[uint64],
 	}
 	return [][32]byte{}, nil
 }
+
+func PayloadProof(ctx context.Context, block *blocks.BeaconBlock) ([][]byte, error) {
```

Comment:
- Let's put this in the same file as the other two. This is the state package.
- This function only makes sense when `block.version >= version.Bellatrix`. Phase0 and Altair forks didn't have a payload. We should check this at the beginning.

Reply:
> * Let's put this in the same file as the other two. This is the state package.
> 
>     * This function only makes sense when `block.version >= version.Bellatrix`. Phase0 and Altair forks didn't have a payload. We should check this at the beginning.

putting this in the blocks package requires me to import `fieldtrie` which causes an import cycle error